src/components/Posts/PostHeader/PostHeader.tsx

```diff
@@ -22,7 +23,7 @@ const PostHeader: FC<PostHeaderProps> = ({ post, page }) => {
   return (
     <>
       <Flex
-        $mt={[40, 72]}
+        $mt={[40, 12]}
```

Comment:
I realise that you didn't start this pattern but we should try get rid of components using margins for their placement on the page.

Reply:
100%! I have refactored this here as it didn't appear to be used anywhere else than the `PostSingleLayout` so hopefully no unexpected side effects 👁️ 👁️